testapps/testapp/src/main/AndroidManifest.xml

```diff
@@ -43,7 +45,8 @@
         <activity
             android:name="com.microsoft.identity.client.testapp.MainActivity"
             android:theme="@style/AppTheme.NoActionBar"
-            android:windowSoftInputMode="stateHidden">
+            android:windowSoftInputMode="stateHidden"
+            android:taskAffinity="">
```

Comment:
What do we expect `""` to do?

Reply:
It sets the task affinity to null.  I can add an HTML comment in the manifest above the activity element to act as a reminder.